applications/DEMApplication/python_scripts/sphere_strategy.py

```diff
@@ -718,6 +718,10 @@ def ModifySubProperties(self, properties, param = 0):
         if not properties.Has(ROLLING_FRICTION_WITH_WALLS):
             properties[ROLLING_FRICTION_WITH_WALLS] = properties[ROLLING_FRICTION]
 
+        if not properties.Has(DEM_ROLLING_FRICTION_MODEL_NAME):
+            properties[DEM_ROLLING_FRICTION_MODEL_NAME] = "DEMRollingFrictionModelConstantTorque"
+            self.Procedures.KratosPrintWarning("Using a default rolling friction model [DEMRollingFrictionModelConstantTorque].")
```

Comment:
Here we could print the properties id, so the message can be understood without debugging

Reply:
Updated.